library/core/src/iter/traits/iterator.rs

```diff
@@ -22,8 +22,8 @@ fn _assert_is_object_safe(_: &dyn Iterator<Item = ()>) {}
 /// generally, please see the [module-level documentation]. In particular, you
 /// may want to know how to [implement `Iterator`][impl].
 ///
-/// [module-level documentation]: index.html
-/// [impl]: index.html#implementing-iterator
+/// [module-level documentation]: crate::iter
+/// [impl]: crate::iter#implementing-iterator
```

Comment:
This is the current module, right?
```suggestion
/// [module-level documentation]: self
/// [impl]: self#implementing-iterator
```

Reply:
Ah, you're right :)